docs/projects/overview.md

```diff
@@ -2,5 +2,5 @@
 
 The SpatialOS GDK for Unity is accompanied by the following projects to help you get started:
 
-* [FPS Starter project]({{urlRoot}}/projects/template-fps/overview)
```

Comment:
that should probably stay at top for now as we don't have a blank project

Reply:
The reason for swapping them was that the quicklink at the bottom of the page was pointing to the blank project page. Will revert the swap.